includes/Modules/Ads/Enhanced_Conversions.php

```diff
@@ -66,8 +71,18 @@ protected function get_user_data() {
 	 * Conditionally enqueues the necessary script for Enhanced Conversions.
 	 *
 	 * @since 1.159.0
+	 * @since n.e.x.t Add the hashed user data to gtag if it exists
```

Comment:
Please end this comment with a period.

```suggestion
	 * @since n.e.x.t Add the hashed user data to the GTag if it exists.
```

This should be enforced by a PHPCS linting rule, to align with our JS linting.

I've made a note to create an issue for this. Alternatively, feel free to raise the issue yourself - please let me know if you do so I can remove it from my to-do list.

Reply:
I created an issue for that #11293